custom_components/contact_energy/sensor.py

```diff
@@ -196,85 +244,15 @@ def make_attribute(self, date, today, response):
-                        statistic_id=f"{DOMAIN}:energy_consumption",
-                        unit_of_measurement=ENERGY_KILO_WATT_HOUR
-                    )
-                    statistics = StatisticData(
-                        start=math.floor(dt_util.as_timestamp(dt_util.as_utc(datetime.strptime(point['date'], '%Y-%m-%dT%H:%M:%S.%f+12:00')))),
-                        sum=point['value']
-                    )
-                    async_add_external_statistics(self.hass, metadata, statistics)'''
             else:
                 _LOGGER.warning('No usage data available for today')
                 # %1', today)
+            
         else:
             _LOGGER.warning('No data available for')
             # %1', today)
         return data
-
-'''
-class ContactEnergyPricesSensor(Entity):
-    def __init__(self, name, api, date_format):
-        self._name = name
-        self._icon = "mdi:account-cash"
-        self._state = 0
-        self._state_attributes = {}
-        self._unit_of_measurement = 'kr'
-        self._date_format = date_format
-        self._api = api
-
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return self._name
-
-    @property
-    def icon(self):
-        """Icon to use in the frontend, if any."""
-        return self._icon
-
-    @property
-    def state(self):
-        """Return the state of the device."""
-        return self._state
-
-    @property
-    def extra_state_attributes(self):
-        """Return the state attributes of the sensor."""
-        return self._state_attributes
-
-    @property
-    def unit_of_measurement(self):
-        """Return the unit of measurement."""
-        return self._unit_of_measurement
-
-    def update(self):
-        """Update state and attributes."""
-        _LOGGER.debug('Checking jwt validity...')
-        if self._api.check_auth():
-            data = self._api.get_price_data()
-            if data:
-                value = data['current_month']['value'] if data['current_month'] else 0
-                self._state_attributes['current_month_value'] = (value / 100) if value != 0 and value != None else 0
-                for condition in MONITORED_CONDITIONS_DEFAULT:
-                    if condition == 'current_month':
-                        self.make_data_attribute(condition, data.get(condition, None), 'cost_in_kr')
-                    elif condition == 'current_price':
-                        self._state = str(data.get(condition, None) / 100)
-                    else:
-                        self._state_attributes[condition] = data.get(condition, None)
-            spot_price_data = self._api.get_spot_price()
-            if spot_price_data:
-                _LOGGER.debug('Fetching daily prices...')
-                today = datetime.now().replace(hour=0, minute=0, second=0, microsecond=0)
-                todaysData = []
-                tomorrowsData = []
+    
                 yesterdaysData = []
```

Comment:
What's happening here? Identation does not match.

```
Logger: homeassistant.config
Source: loader.py:737
First occurred: 2:12:55 PM (1 occurrences)
Last logged: 2:12:55 PM

Platform error: sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 728, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 745, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/contact_energy/sensor.py", line 256
    yesterdaysData = []
IndentationError: unexpected indent

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config.py", line 923, in async_process_component_config
    platform = p_integration.get_platform(domain)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 737, in get_platform
    raise ImportError(
ImportError: Exception importing custom_components.contact_energy.sensor

```

Reply:
Sorry, during getting this to work I removed all this commented out code, and just before I contacted you I put it back in. but it looks like I only copied half of it. My bad